src/services/datahub-client.ts

```diff
@@ -199,23 +203,25 @@ export class HttpDatahubClient implements DatahubClient {
     }
   }
 
-  public async getDailyVisitsByPeriod({
+  public async getDailyAndHoursVisitsByPeriod({
     domainName,
     dateFrom,
+    periodByhours,
   }: {
     domainName: string;
     dateFrom: Date;
-  }): Promise<DailyVisitsResult> {
+    periodByhours: boolean;
+  }): Promise<DailyAndHoursVisitsResult> {
     try {
       const response = await this.customerGet<any>(
         `domains/${domainName}/events/quantity-summarized-by-period`,
         {
           startDate: dateFrom.toISOString(),
-          periodBy: 'days',
+          periodBy: periodByhours ? 'hours' : 'days',
```

Comment:
What do you think about exposing it as a union type and accept it as a parameter? you can also make one of these values as default using deconstruction:

```ts
type SummaryPerio = 'hours' | 'days';

// . . .

  public async getVisitsByPeriod({
    domainName,
    dateFrom,
    periodBy = 'days',
  }: {
```

Also `getVisitsByPeriod` seems to be more clear.